schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+/* Database schema to keep the structure of entire database. */
```

Comment:
- [ ] all FOREIGN KEYS should have indexes created. please add them.

Reply:
Hi @omar-labana, thank you for the code review.